src/JSONNestedNode.js

```diff
@@ -89,15 +89,21 @@ const STYLES = {
 export default class JSONNestedNode extends React.Component {
   static defaultProps = {
     data: [],
+    nodeExpanded: false,
```

Comment:
`false` is confusing - I think it's better to use `null` or remove default prop here, since it's already defined in `src/index.js`